<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>blackJack_QTClass</class>
 <widget class="QMainWindow" name="blackJack_QTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>blackJack_QT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="player1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>210</width>
      <height>256</height>
     </rect>
    </property>
    <property name="title">
     <string>PLAYER</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="p1_c1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>125</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p1_c2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>95</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p1_c3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>65</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p1_c4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>35</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p1_c5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>5</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pointLabel1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="player3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>420</y>
      <width>210</width>
      <height>256</height>
     </rect>
    </property>
    <property name="title">
     <string>NPC P3</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="p3_c1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>125</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p3_c2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>95</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p3_c3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>65</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p3_c4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>35</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p3_c5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>5</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pointLabel3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Bank">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>210</width>
      <height>256</height>
     </rect>
    </property>
    <property name="title">
     <string>BANK</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="bank_c1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>125</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bank_c2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>95</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bank_c3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>65</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bank_c4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>35</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bank_c5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>5</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pointLabel4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="player2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>210</width>
      <height>256</height>
     </rect>
    </property>
    <property name="title">
     <string>NPC P2</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="p2_c1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>125</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p2_c2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>95</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p2_c3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>65</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p2_c4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>35</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p2_c5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>5</y>
       <width>90</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pointLabel2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>430</y>
      <width>40</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="pixmap">
        <pixmap>image/coin3.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="playerCoinLabel2">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>430</y>
      <width>40</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="pixmap">
        <pixmap>image/coin3.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="playerCoinLabel3">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>210</y>
      <width>76</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>   50 coin</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string> 100 coin</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string> 200 coin</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>40</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="pixmap">
        <pixmap>image/coin3.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="playerCoinLabel1">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="compareLabel1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="compareLabel2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="compareLabel3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>340</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="blackJack_QT.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>addCard()</slot>
  <slot>test()</slot>
  <slot>testRB()</slot>
 </slots>
</ui>
